docs/source/Evaluation_Tutorial.md

```diff
@@ -26,145 +26,79 @@ Once you know which dataset you want to train on, you can follow the instruction
 
-> Alternatively, you can use the same docker image on your own container-orchestration system following similar steps.
+You can use our docker image on your own container-orchestration system by following similar steps.
+
+There are a few options for model training depending on the dataset you choose to train on:
+
+* [Train on UnityGroceries-Synthetic dataset](#train-on-unitygroceries-synthetic-dataset)
+* [Train on UnityGroceries-RealWorld dataset](#train-on-unitygroceries-realworld-dataset)
+* [Train on synthetic dataset generated on Unity Simulation](#train-on-synthetic-dataset-generated-on-unity-simulation)
 
 ### Train on UnityGroceries-Synthetic dataset
 
 This section shows you how to train a model on the sample synthetic dataset. Note that this is a small dataset which is the fastest to train but won't produce the best results; for that, you can train a model that uses a larger synthetic dataset and [fine tunes the model on real images](#train-on-synthetic-and-real-world-dataset-optional). To observe the best results we have obtained, you can follow the instructions to run one of our [pre-trained models](#using-our-pre-trained-models) below.
 
 To train the model, simply import [**this pre-compiled pipeline**](https://raw.githubusercontent.com/Unity-Technologies/datasetinsights/master/kubeflow/compiled/train_on_synthdet_sample.yaml) into your kubeflow cluster. The figure below shows how to do this using the [web UI](https://www.kubeflow.org/docs/pipelines/pipelines-quickstart/#deploy-kubeflow-and-open-the-pipelines-ui). You can optionally use the [KFP CLI Tool](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool).
 
-![upload pipeline](_images/kubeflow/upload_pipeline.png)
+<img src="_images/kubeflow/upload_pipeline.png" alt="drawing" width="600"/>
 
 Once your pipeline has been imported, you can run it via the web UI as shown below. Alternatively, you can use the [KFP CLI Tool](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool)
-![train on SynthDet sample](_images/kubeflow/train_on_synthdet_sample.png)
+
+<img src="_images/kubeflow/train_on_synthdet_sample.png" alt="drawing" width="600"/>
 
 You have to specify run parameters required by this pipeline:
 
 - `docker`: Path to a Docker Registry. We suggest changing this parameter to pull our images on Docker Hub with a specific tag, such as `unitytechnologies/datasetinsights:0.2.0`
 - `source_uri`: The dataset source uri. You can use the default value which points to the required dataset for this pipeline.
 - `config`: Estimator config YAML file. You can use the default value which points to a YAML file packaged with our docker images.
-- `tb_log_dir`: Path to store tensorboard logs used to visualize the training progress.
-- `checkpoint_dir`: Path to store output Estimator checkpoints. You can use one of the checkpoints for estimator evaluation.
+- `tb_log_dir`: GCS path to store tensorboard logs used to visualize the training progress.
```

Comment:
This is actually intentionally vague. The current implementation works for both GCS path and local path. If users are interested to use their own container orchestration system, they can mount a persistent storage of this path and read from this path if desired, as long as the path to such file is fully specified. 

Although I said that, it's currently not trivial to run this under their own container orchestration system. These pre-compiled pipeline only works for GCP/GKE kubeflow.

Reply:
Fixed